abci/versions.go

```diff
@@ -81,3 +108,15 @@ func (v Version) GetStartArgs(args []string) []string {
 		"--transport=grpc",
 	)
 }
+
+// ensureUniqueNames checks for duplicate names in a slice of Versions.
+func ensureUniqueNames(versions []Version) error {
```

Comment:
let's maybe call that `func (v Versions) Validate() error` so we can call that in passthrough too.